android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml

```diff
@@ -110,6 +110,31 @@
                     android:text="@string/projects_credits_host_and_user"
                     android:textAppearance="?android:attr/textAppearanceSmall"
                     android:textColor="?android:attr/textColorPrimary"/>
+            </LinearLayout>
+
+            <LinearLayout
```

Comment:
Please fix indents

Reply:
Please look at whole file. This is the correct indent

Reply:
It's still not correct:
![image](https://user-images.githubusercontent.com/3234209/152434973-642086b3-cc9d-4d2b-b249-7202c8b2f6c9.png)

Reply:
ok. I will fix it. Thank you for noticing